### ISSUE TITLE
Couple of visual bugs

### ISSUE THREAD

**Issue by sieglinde** [Fausto-Korpsvart/Material-GTK-Themes#9]
Strange looking right click menu on Gnome Terminal and current workspace text barely visible.
![VirtualBox_Cinnamon_17_07_2023_01_07_47](https://github.com/Fausto-Korpsvart/Material-GTK-Themes/assets/5104577/854a8739-f229-4a14-a1de-3417cb89aaa1)
![VirtualBox_Cinnamon_17_07_2023_01_08_05](https://github.com/Fausto-Korpsvart/Material-GTK-Themes/assets/5104577/e61acd64-147b-4f1e-9752-d8459afa4a62)

**Comment by M3dvidek** [comment]
Hi,
I have the same problems in gnome applications like extentions or tweaks on gnome 46.4

**Comment by Fausto-Korpsvart** [comment]
> Hi, I have the same problems in gnome applications like extentions or tweaks on gnome 46.4

Sorry, this theme is not yet updated to newer versions of Gnome, I hope that for Gnome 47 I will have enough time to update it.

**Comment by sieglinde** [comment]
Are you still planning to update this? This is a very good theme!

**Comment by Fausto-Korpsvart** [comment]
> Are you still planning to update this? This is a very good theme!

Yes, but I've been busy with my work lately and I haven't had much time, the theme has already started, there are still details missing that I will finish when I have a little time.
